Kernel/Bus/USB/USBTransfer.h

```diff
@@ -9,13 +8,13 @@
 #include <AK/AtomicRefCounted.h>
 #include <AK/OwnPtr.h>
 #include <Kernel/Bus/USB/PacketTypes.h>
+#include <Kernel/Bus/USB/USBDMAPool.h>
```

Comment:
Unclear why this include is needed in the second commit. Also the re-formatting of the copyright looks incorrect here.